src/generate-elements.ts

```diff
@@ -85,7 +86,7 @@ function serializeElement(
   const events = resolvedElement.events.map(
       e => ({
         name: e.name,
-        description: e.description,
+        description: e.description || '',
```

Comment:
Before null-tracking I would say that I prefer non-null fields so that client code doesn't have to branch too many times to deal with null. I still think this is true, especially with collections. If the primary client branching would come from detecting empty though, it might not buy much, and with null-tracking you at least get warnings.

So `null` could make sense here. I'm not so sure about `undefined` as the 'not present' value. Here I'd like to represent that the object has a description property, that may be null, rather than not having a description property at all. Of course, to be maximally confusing, setting a property to `undefined` in JS means that the object has an own property name, but it definitely clouds the issue.